lib/daru/io/exporters/excel.rb

```diff
@@ -11,7 +11,23 @@ class Excel < Base
         # @param dataframe [Daru::DataFrame] A dataframe to export
         # @param path [String] Path of the file where the +Daru::DataFrame+
         #   should be written.
-        # @param options [Hash] A set of options containing user-preferences
+        # @param options [Hash<Hash>] A +Hash+ containing +:display+ and +:formatting+ keys.
+        #
+        # @option options display [Hash<Boolean>] A Hash of display options. Supported keys are
+        #   +:header+, +:data+ and +:index+. Default value of each key is set to +true+. For
+        #   example, if +:header+ is set to true, the headers are written.
+        #
+        # @option options formatting [Hash<Hash>] A Hash of formatting options. Supported keys
+        #   are +:header+, +:data+ and +:index+. For example, if +:header+ is set to a Hash
+        #   containing keys such as +:color+, the headers are written with the specified color.
```

Comment:
Acknowledged, this does seem much more intuitive to use. Made this and the below changes with commit 6fe329a.